pcsx2/CDVD/CsoFileReader.cpp

```diff
@@ -70,10 +71,10 @@ bool CsoFileReader::CanHandle(const std::string& fileName, const std::string& di
 
 bool CsoFileReader::ValidateHeader(const CsoHeader& hdr, Error* error)
 {
-	if (hdr.magic[0] != 'C' || hdr.magic[1] != 'I' || hdr.magic[2] != 'S' || hdr.magic[3] != 'O')
+	if ( (hdr.magic[0] != 'C' && hdr.magic[0] != 'Z' ) || hdr.magic[1] != 'I' || hdr.magic[2] != 'S' || hdr.magic[3] != 'O')
```

Comment:
Extra space before ( ? I guess that wasn't intended :)

Reply:
Fixed.